ui/src/app/app-routing.tsx

```diff
@@ -93,7 +95,7 @@ interface RoutingProps {
   signIn: () => void;
 }
 
-export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSignIn, signIn}) => {
+export const AppRoutingComponentImpl: React.FunctionComponent<RoutingProps> = ({onSignIn, signIn}) => {
```

Comment:
why the rename? I do see some other usages of the `Impl` suffix in the UI codebase but it seems like it's usually as a private variable so that the non suffixed version can be exported.

Ex. `export const AdminWorkspaceSearch = withUrlParams()(AdminWorkspaceSearchImpl);`

Reply:
I can wrap the whole component, it'll just look a bit ugly the way it's currently phrased.